package/gluon-mesh-vpn-tunneldigger/files/usr/bin/tunneldigger-watchdog

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/lua
+
+local uci = require('simple-uci').cursor()
+
+function restart_tunneldigger()
+	io.popen('logger -t tunneldigger-watchdog "Restarting Tunneldigger."')
```

Comment:
Since we don't parse any return values we might use `os.execute` here as well, no?